lib/linguist/heuristics.yml

```diff
@@ -110,7 +110,7 @@ disambiguations:
     # target : \
     #  : dependency
     # path/file.ext1 : some/path/../file.ext2
-    pattern: '([\/\\].*:\s+.*\s\\$|: \\$|^ : |^[\w\s\/\\.]+\w+\.\w+\s*:\s+[\w\s\/\\.]+\w+\.\w+)'
+    pattern: '([\/\\].*:\s+.*\s\\$|: \\$|^[ %]:|^[\w\s\/\\.]+\w+\.\w+\s*:\s+[\w\s\/\\.]+\w+\.\w+)'
```

Comment:
/cc @MartinNowak, who wrote this regular expression, in case they want to review this change.

Reply:
For anyone reviewing this, the sample that was not catched here was `samples/Makefile/makefile`:

```make
#!/usr/bin/make -f
%:
	ls -l
```